scripts/run-deploy.js

```diff
@@ -0,0 +1,12 @@
+const { evergreenDeploy, localDeploy } = require("./deploy-production.js");
+
+const main = async () => {
+  if (process.argv.includes("--local")) {
+    await localDeploy();
+    return;
+  }
+  await evergreenDeploy();
+  return;
```

Comment:
nit. I got a linter warning for an unnecessary return.

Reply:
Do you know what rules you have applied locally that are not covered by Spruce's lint config? It'd be nice to have the two standardized.

Reply:
Same comment about this being a default rule in the airbnb config we are using. https://github.com/airbnb/javascript/blob/fd77bbebb77362ddecfef7aba3bf6abf7bdd81f2/packages/eslint-config-airbnb-base/rules/best-practices.js#L363
I'm unsure why it isn't flagging in CI or for you locally.